### DISCUSSION TITLE
Rebuild App stucture according to Flutter 3.13.4

### DISCUSSION THREAD

**Discussion by bhargavraviya** [nylo-core/nylo#74] [General] [answered]
@agordn52 

I am interested in working with you. Please guide me, starting with small tasks (good first issue).

You gave me permission to install fresh Flutter version 3.13.4 and merge it to the Nylo project according to your project setup.

because Flutter developers often change the native code to not refer to your reps and other things.

**Comment by agordn52** [comment]
Hi @bhargavraviya , I'll create a new discussion with some ideas 💡

**Reply by agordn52** [reply, marked as answer]
Hi @bhargavraviya,

I've created a new discussion [here](https://github.com/nylo-core/nylo/discussions/78) 👍
